lib/Dialect/FIRRTL/Export/FIREmitter.cpp

```diff
@@ -645,14 +685,45 @@ void Emitter::emitPrimExpr(StringRef mnemonic, Operation *op,
   os << ")";
 }
 
+void Emitter::emitAttribute(MemDirAttr attr) {
+  switch (attr) {
+  case MemDirAttr::Infer:
```

Comment:
I think that ODS puts out an "enum to string" function into a generated .cpp.inc file, can you use that?

Reply:
I tried it.. unfortunately the `stringifyMemDirAttr()`function  et cetera are printing the enum name with an uppercase first letter, and in the case of `RUW` prints `ReadWrite` instead of `rdwr`.